app/public/css/styles.css

```diff
@@ -1646,6 +1647,29 @@ button[data-balloon] {
   margin-right: 16px;
 }
 
+.buy-ada {
+  margin-top: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.buy-ada-link,
```

Comment:
Extract this to a separate class that can be used on any button, ie. `.button.link`

Reply:
Fixed